test/e2e/utils/asserts.go

```diff
@@ -16,3 +20,9 @@ func ExpectMaybeNotFound(err error) {
 		panic(err.Error())
 	}
 }
+
+func ExpectNotFound(err error) {
+	if err == nil || !errors.IsNotFound(err) {
```

Comment:
Is it correct error panic when `err == nil`?

Reply:
Probably not and the method was never used in the end, so I have just removed.